bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/ghost/DetectorManufacturer.java

```diff
@@ -0,0 +1,43 @@
+package edu.gemini.spModel.gemini.ghost;
+
+public enum DetectorManufacturer {
+    BLUE("E2V_CCD231-84-1-G57", "Blue", DetectorManufacturer.E2V_PIXEL_SIZE, 4096, 4112, 0.0003263888888888889),
+    RED ("E2V_CCD231-C6-1-G58", "Red", DetectorManufacturer.E2V_PIXEL_SIZE, 6144, 4096, 0.00022916666666666666);
```

Comment:
The dark current values are extremely precise?  I wonder if we need to use `BigDecimal`s for them.